Squeezer/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     compile 'com.google.code.findbugs:jsr305:2.0.2'
 
     // EventBus, https://github.com/greenrobot/EventBus.
-    compile project(':libs:eventbus:EventBus')
+    compile 'de.greenrobot:eventbus:2.4.1'
```

Comment:
Just did a few more tests based on the [faq ](http://greenrobot.org/eventbus/documentation/faq/)mentioned in [issue 149](https://github.com/greenrobot/EventBus/issues/149) . 

> The exception is thrown if the class has methods with a parameter that is unavailable to the API level of the device.

When I launch squeezer on an API 7 (Android 2.1) emulator I get a crash on line 81 of SubscriberMethodFinder.java, which makes sense as this is the fallback ("plan b") applied by EventBus when getDeclaredMethods fails.

I then comment out the `Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP` section of updateOngoingNotification and the crash disappears.

I suspect (but I'm guessing here as I haven't done the test) that the crash would happen also with the EventBus fork pulled in and on EventBus 3.0 (unless we use an index).